src/workerd/io/worker.c++

```diff
@@ -1258,20 +1258,11 @@ Worker::Worker(kj::Own<const Script> scriptParam,
     }
   });
 
-  auto maybeMakeSpan = [&](auto operationName) -> MaybeSpan {
-    if (auto span = systemTracer.makeSpan(operationName)) {
-      span.setTag("truncated_script_id"_kj, truncateScriptId(script->getId()));
-      return kj::mv(span);
-    } else {
-      return MaybeSpan();
-    }
-  };
-
-  MaybeSpan currentSpan = maybeMakeSpan("lw:new_startup_metrics"_kj);
+  auto currentSpan = parentSpan.newChild("lw:new_startup_metrics"_kj);
```

Comment:
Looks like these are missing the truncated script ID tag. Though maybe that tag shouldn't exist on every span and should really just exist on the parent span.

Reply:
Whoops, good catch.

Looks like the internal tests would have caught this, I probably should have gotten them running before marking this ready for review.

Reply:
> Though maybe that tag shouldn't exist on every span and should really just exist on the parent span.

The intent was for it to exist on as many spans as possible, since it's hard to query for spans which are descendants of other spans which have specific tags.